### PR TITLE
Pyramid update - final

### PR DESCRIPTION
**Pull request by prycrane**
**Trello**

- https://trello.com/c/wDtdrXVX

**Preview**

- https://deploy-preview-273--systeme-conception-canada-ca.netlify.app/guide-redaction/#wp2-1

**Design document**

- https://docs.google.com/document/d/1iB6qJslVdQrz3XGSFjx62hEWlXld_uQUkkUVmSMgd6s/edit#heading=h.n46usj4t10d0

**English PR**

- https://github.com/canada-ca/design-system/pull/397

### CODE REVIEW

File: guide-redaction/index.md [hunk 1]
```diff
@@ -234,38 +234,40 @@ title: "Guide de rédaction du contenu du site Canada.ca"
...
+        </ul>
+      </li>
+      <li><strong>Plus d'information</strong>&nbsp;:
+        <ul>
+          <li>inclure des renseignements pertinents à l'intention des publics cibles qui désirent en apprendre davantage ou qui ont une connaissance approfondie du sujet ou du service.</li>
+        </ul>
+      </li>
+      <li><strong>Liens connexes</strong>&nbsp;:
+        <ul>
+          <li>ajouter des liens seulement s'ils concernent le contenu et appuient la tâche.</li>
```
Comment (donohuech) [inline guide-redaction/index.md:267]: This is missing the parentheses and info inside it at the end, the full bullet point should be: 

ajouter des liens seulement s'ils concernent le contenu et appuient la tâche (par exemple, des lois ou des publications).

File: guide-redaction/index.md [hunk 2]
```diff
@@ -234,38 +234,40 @@ title: "Guide de rédaction du contenu du site Canada.ca"
...
-      <figcaption class="text-center"> <strong>Figure 1</strong> </figcaption>
-      <img alt="Pyramide inversée illustrant comment créer une page Web en indiquant tout d'abord les renseignements clés." class="img-responsive center-block" id="fig1" src="/guide-redaction/images/fig-01-fra.jpg">
-      <details class="brdr-tp brdr-rght brdr-bttm brdr-lft">
-        <summary class="wb-toggle" data-toggle="'{&quot;print&quot;:&quot;on&quot;}'">Figure 1 - Version textuelle</summary>
-        <p>Pyramide inversée illustrant comment créer une page Web en indiquant tout d'abord les renseignements clés.</p>
+    <div class="row mrgn-tp-lg mrgn-bttm-md">
+      <div class="col-md-8">
+        <figure><img alt="Pyramide inversée illustrant comment créer une page Web en indiquant tout d'abord les renseignements clés." class="img-responsive" id="fig1" src="/guide-redaction/images/pyramide-fr.png" />
+          <figcaption>
+            <p><strong>Figure 1:</strong> pyramide inversée pour l’organisation du contenu</p>
```
Comment (donohuech) [inline guide-redaction/index.md:241]: This should have a space between 'Figure 1' and the colon. So: Figure 1 : pyramide....

**Review by donohuech — COMMENTED** [review]
Two comments on lines within the text. We also need to clarify why image text and heading text are different for 'tâches prioritaires'

File: guide-redaction/index.md (hunk 1 above)
Comment (prycrane) [reply guide-redaction/index.md]: Done

File: guide-redaction/index.md (hunk 2 above)
Comment (prycrane) [reply guide-redaction/index.md]: Done

**Pull request merged**